features/admin_journey.feature

```diff
@@ -1,3 +1,4 @@
+@not-production
```

Comment:
Is it possible to tag individual scenarios, or only Features?  It would be nice for the Admin to at least log in and log out to check that the production admin app is up.